src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -85,28 +88,29 @@ internal virtual void PackFromXyzBytes(BufferSpan<byte> sourceBytes, BufferSpan<
         /// <summary>
         /// Bulk version of <see cref="IPixel.ToXyzBytes(byte[], int)"/>.
         /// </summary>
-        /// <param name="sourceColors">The <see cref="BufferSpan{T}"/> to the source colors.</param>
-        /// <param name="destBytes">The <see cref="BufferSpan{T}"/> to the destination bytes.</param>
+        /// <param name="sourceColors">The <see cref="Span{T}"/> to the source colors.</param>
+        /// <param name="destBytes">The <see cref="Buffer{T}"/> to the destination bytes.</param>
+        /// <param name="startIndex">The starting index of the <paramref name="destBytes"/>.</param>
         /// <param name="count">The number of pixels to convert.</param>
-        internal virtual void ToXyzBytes(BufferSpan<TPixel> sourceColors, BufferSpan<byte> destBytes, int count)
+        internal virtual void ToXyzBytes(Span<TPixel> sourceColors, Buffer<byte> destBytes, int startIndex, int count)
```

Comment:
@tocsoft the problem is with the [`IPixel.To***Bytes()` API-s](https://github.com/JimBobSquarePants/ImageSharp/blob/master/src/ImageSharp/PixelFormats/IPixel.cs#L59). They consume an array now.
I think we definitely need to change that, but this is a more complex topic, outside the scope of this refactor.
Temporally it can be `void ToXyzBytes(Span<byte> bytes, int startIndex)`, but I have an idea for a better solution :smile: